repour/scm/git_provider.py

```diff
@@ -105,7 +105,7 @@ def push_with_tags(dir, branch, remote="origin", tryAtomic=True):
         def do(atomic):
             yield from expect_ok(
                 cmd=["git", "push"] + (["--atomic"] if atomic else []) + ["--follow-tags", remote, branch],
-                desc="Could not" + (" atomic" if atomic else "") + " push tag+branch with git",
+                desc="Could not" + (" atomic" if atomic else "") + " push tag+branch with git. Make sure you have the right permissions!",
```

Comment:
The permission issue typically happens, if he haven't added the pnc user to his existing gerrit repository. The question is: Do we want to hardcode it to the error message? I'd vote for yes.